### PR TITLE
Lumen

### PR DESCRIPTION
**Pull request by jxxviel-rgb**
## What does this PR do?
Add resource(s)  | Add info |

### Description
Lumen is the perfect solution for building Laravel based micro-services and blazing fast APIs. In fact, it's one of the fastest micro-frameworks available. It has never been easier to write stunningly fast services to support your Laravel applications.
### Why is this valuable (or not)?
Decidedly Laravel. Delightfully Minimal.
Lightning fast micro-services and APIs delivered with the elegance you expect.
### How do we know it's really free?
on the official page, the author explains that this ebook is given for free.
### For book lists, is it a book? For course lists, is it a course? etc.
ebook
## Checklist:
- [O] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [O] Search for duplicates.
- [O] Include author(s) and platform where appropriate.
- [O] Put lists in alphabetical order, correct spacing.
- [O] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-id.md [hunk 1]
```diff
@@ -126,6 +127,10 @@
 * [Tutorial Pemrograman PHP untuk Pemula](https://www.petanikode.com/tutorial/php) - Ahmad Muhardian (Petani Kode) *(Dalam Proses)*
 
 
+### Lumen
```
Comment (davorpa) [inline books/free-programming-books-id.md:130]: ℹ️ It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.

⚠️ Preserve alphabetization of list sections

File: books/free-programming-books-id.md [hunk 2]
```diff
@@ -126,6 +127,10 @@
...
+
+* [Lumen Fundamental](https://santrikoding.com) - Yudi Purwanto(HTML) [(PDF)](https://santrikoding.com/account/ebook/lumen-fundamental) _(Membutuhkan akun santrikoding atau email yang valid)_
```
Comment (davorpa) [inline books/free-programming-books-id.md:132]: One link is enough, use the most specific and select right format: HTML or PDF

See also CONTRIBUTE format guidelines to solve lint errors

**Review by davorpa — CHANGES_REQUESTED** [review]
Could you add description about what is **Lumen** and why is valuable?

File: books/free-programming-books-id.md (hunk 1 above)
Comment (jxxviel-rgb) [reply books/free-programming-books-id.md:130]: lumen is micro-framework based on laravel, especially building micro-services and blazing fast APIs.
do i put it in laravel category or leave it in lumen category?

Comment (davorpa) [reply books/free-programming-books-id.md:113]: If can't be found others others enough... then I agree with Laravel.

There are seven lifes to be always improving 😉.

**Comment by davorpa** [comment]
😲 Note about lint faults. OVERSCOPED

ℹ Configure your vscode/editor workspace to avoid regenerate TOC and have `*` as markdown list token 😉

When all will be ready, request for review again (see buttons near this github section)

File: books/free-programming-books-id.md [hunk 3]
```diff
@@ -10,6 +10,7 @@
 * [HTML & CSS](#html-css)
 * [Java](#java)
 * [JavaScript](#javascript)
+* [Laravel](#laravel)
```
Comment (davorpa) [inline books/free-programming-books-id.md:13]: It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.

File: books/free-programming-books-id.md [hunk 4]
```diff
@@ -87,6 +88,11 @@
 * [Tutorial Dasar Javascript untuk Pemula](https://www.petanikode.com/tutorial/javascript/) *(Dalam Proses)*
 
 
+### Laravel
+
+* [Lumen Fundamental](https://santrikoding.com) - Yudi Purwanto(HTML) [(PDF)](https://santrikoding.com/account/ebook/lumen-fundamental) _(Membutuhkan akun santrikoding atau email yang valid)_
```
Comment (davorpa) [inline books/free-programming-books-id.md:93]: A bit of help according previous review

```suggestion
* [Lumen Fundamental](https://santrikoding.com/ebook/lumen-fundamental) - Yudi Purwanto (PDF) *(Membutuhkan akun santrikoding atau email yang valid)*
```

**Review by davorpa — CHANGES_REQUESTED** [review]
Since Laravel is PHP Framework and no more resources was found... **it fits into `PHP` section**

File: books/free-programming-books-id.md (hunk 4 above)
Comment (eshellman) [reply books/free-programming-books-id.md:93]: I don't think we can accept a link that points to a login page with no information about the book we want to link to.

**Comment by eshellman** [comment]
Sorry, we need to decline this one, thanks for the suggestion.

**Pull request closed without merging**